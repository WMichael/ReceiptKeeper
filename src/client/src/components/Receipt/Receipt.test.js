import React from 'react';
import Receipt from './Receipt';
import {shallow} from 'enzyme';

describe('<Receipt>', () => {
  let wrapper;
  
  // beforeEach(() => {
  //   wrapper = shallow(<Receipt />);
  // });

  // test('Renders correctly', () => {
  //   expect(wrapper).toMatchSnapshot();
  // });
});