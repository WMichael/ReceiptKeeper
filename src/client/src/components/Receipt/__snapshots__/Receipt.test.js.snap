// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Receipt> Renders correctly 1`] = `undefined`;
